api/handlers_images.go

```diff
@@ -92,3 +93,53 @@ func (s *server) ImageGetHandler(w http.ResponseWriter, r *http.Request) {
 
 	handleResponseOk(w, toEc2ImageResponse(out[0]))
 }
+
+func (s *server) ImageUpdateHandler(w http.ResponseWriter, r *http.Request) {
+	w = LogWriter{w}
+	vars := mux.Vars(r)
+	account := s.mapAccountNumber(vars["account"])
+	id := vars["id"]
+
+	req := &Ec2ImageUpdateRequest{}
+	if err := json.NewDecoder(r.Body).Decode(req); err != nil {
+		handleError(w, err)
```

Comment:
You need something like this here, otherwise it returns a 500
```
		msg := fmt.Sprintf("cannot decode body into update image input: %s", err)
		handleError(w, apierror.New(apierror.ErrBadRequest, msg, err))
```